src/lib.rs

```diff
@@ -1,5 +1,55 @@
 //! Winit is a cross-platform window creation and event loop management library.
 //!
+//! # Installation
+//!
+//! `winit` can be added to `Cargo.toml` as a dependency. It can be added via `cargo add`.
+//!
+//! ```bash
+//! $ cargo add winit
+//! ```
+//!
+//! To only enable the X11 backend on Free Unix[^unix] systems, disable default features
+//! and enable the `x11` feature.
+//!
+//! ```bash
+//! $ cargo add winit --no-default-features --features x11
+//! ```
+//!
+//! To only enable the Wayland backend on Free Unix systems, disable default features
+//! and enable the `wayland` feature.
+//!
+//! ```bash
+//! $ cargo add winit --no-default-features --features wayland
+//! ```
+//!
+//! These features have no effect on systems that are not Free Unix.
+//!
+//! ## Dependencies
+//!
+//! Dependencies on non-system libraries is managed through Cargo. For the X11
+//! backend, the following Ubuntu packages or their equivalents must be installed.
```

Comment:
They are not a `must`, though it depends on a feature you're using. you also list ubuntu packages and not the libraries themselves, which is wrong. You should list libraries, I guess.

We also don't really use `-dev` for `dlopen` way of running things.

Reply:
> you also list ubuntu packages and not the libraries themselves, which is wrong. You should list libraries, I guess.

Is there a better way to identify these libraries?

Reply:
IIRC if you build without `dlopen`, you can just `lddtree` it.